types/power-assert/index.d.ts

```diff
@@ -62,6 +62,8 @@ declare namespace assert {
 
     export function ifError(value:any):void;
 
+    export const strict: typeof assert;
```

Comment:
Do you need to update the version to 1.5?

Reply:
Yes, we need to update the version to 1.5 to use this API.

Reply:
Can you update the version in the header to reflect that. After you've done that I'll merge the change.

Reply:
Okay, done.